rosdep/python.yaml

```diff
@@ -5601,6 +5601,10 @@ python3-backoff-pip:
   ubuntu:
     pip:
       packages: [backoff]
+python3-bidict:
+  debian: [python3-bidict]
+  fedora: [python3-bidict]
+  ubuntu: [python3-bidict]
```

Comment:
Same goes for Ubuntu.  This is only in Ubuntu since Focal, and we still support Xenial and Bionic:

```
ubuntu:
  '*': [python3-bidict]
  bionic: null
  xenial: null
```